vllm/v1/attention/ops/kvarn_store.py

```
@@ -0,0 +1,340 @@
+        sinkhorn_iters: log-domain iterations.
+
+    Returns dict with packed cache record:
+        q_packed_uint8 : ``[D, group/2]`` uint8 — 4-bit pairs (low=even, high=odd)
+        s_col_K        : ``[D]``        fp16   — absorbed per-channel scale
+        zp_K           : ``[D]``        fp16   — absorbed per-channel zero
+        s_row_K        : ``[group]``    fp16   — per-token-in-tile sinkhorn scale
+    """
+    if not 1 <= bits <= 8:
+        raise ValueError(f"bits must be in [1, 8], got {bits}")
+    tile = k_tile_rotated.float()
+    D, G = tile.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `D` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)